apps/examples/testcase/ta_tc/mqtt/itc/itc_mqtt_main.c

```diff
@@ -0,0 +1,319 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <netdb.h>
+#include <apps/shell/tash.h>
+#include <mqtt/mqtt_api.h>
```

Comment:
it causes a build error. right path is <network/mqtt/mqtt_api.h> .

Reply:
@pillip8282  Updated the request. Please check